File: Frontend/DisasterDispatchFrontend/src/app/Models/disaster-operation.ts
import { Guid } from "guid-typescript";
import { DisasterCategory } from "./disaster-category";
import { AppUser } from "./app-user";
import { CustomOperation } from "./custom-operation";
import { BaseEntity } from "./base-entity";

export class DisasterOperation extends BaseEntity {
    disasterCategory:DisasterCategory
    disasterCategoryId:Guid;
    provinceId:string;
    Assigner:AppUser;
    assignerId:string;
    Operations:CustomOperation[];
}

File: Backend/DisasterDispatch.Core/Entities/DisasterOperation.cs
using DisasterDispatch.Core.Entities.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DisasterDispatch.Core.Entities
{
    public class DisasterOperation:BaseEntity
    {
        public DisasterCategory DisasterCategory{ get; set; }
        public Guid DisasterCategoryId { get; set; }
        public string ProvinceId{ get; set; }        
        public AppUser Assigner { get; set; }
        public string AssignerId { get; set; }
        public ICollection<CustomOperation> Operations{ get; set; }

    }
}
